examples/blazor/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
+   // Use hover for the description of the existing attributes
+   // For further information visit https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md
+   "version": "0.2.0",
+   "configurations": [
```

Comment:
I updated the configuration and some issues were resolved (and json much improved) but there are a couple of issues I wanted to ask you about. Bear in mind that I'm using VSCode on a Mac:

* The configuration doesn't appear to work at all with "browser" set to "edge". I get all sorts of errors. Has that been tested with this new blazorwasm configuration? Are there additional settings when configuring the browser?
* Whenever I edit a razor file I have to completely stop and restart the app to see any sort of change. That's rather painful for development. Is there some configuration that can be added so that when running in debug mode it watches for changes, rebuilds, and refreshes automatically?

Reply:
> Whenever I edit a razor file I have to completely stop and restart the app to see any sort of change

If you're not using VS for Windows, the recommended way to run is via `dotnet watch run` on the command line. That will automatically recompile on each change. It doesn't auto-refresh your browser window but whenever you "reload" the page you'll get the latest build of your app.

Reply:
Interestingly, my browser does seem to autorefresh. It's using my system default browser which is Edge, so that's nice too. However, looks like I probably need to handle debugging differently when running that way. Probably need another launch task for attaching to an existing process...

Reply:
Looks like you have to run it one way for auto-refresh and another way for debugging. Does that sound right?

Reply:
Correct -- it's not ideal but it's something we'll be making better in .NET 6.